### PR TITLE
Page - Educational

### LINKED ISSUE #299: Page - Educational
Convert it into mdx file

image

**Figure: Showing SSW Website Consulting page - Educational**

### PR DESCRIPTION
**Pull request by jimmidier**
Closes #299 

image
Route: /educational

### CODE REVIEW

**Review by bradystroud — CHANGES_REQUESTED** [review]
Not a fan of how this button has the icons.
Only links should have those icons

image
**Figure: Icons look out of place**

I understand these icons are on the old website, but it looks like a bug.

**Comment by jimmidier** [comment]
> Not a fan of how this button has the icons. Only links should have those icons
> 
> image **Figure: Icons look out of place**
> 
> I understand these icons are on the old website, but it looks like a bug.

Done - Removed the icons. Please re-review

File: public/images/icons/IconPdf.png
Comment (bradystroud) [inline public/images/icons/IconPdf.png:1]: Remove if not used

File: public/images/icons/arrow-right.png
Comment (bradystroud) [inline public/images/icons/arrow-right.png:1]: Remove if not used

File: pages/educational/index.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+            height={0}
+            alt="SSW Educational Banner"
+            sizes="100vw"
+          />
+        </DownloadWhitepaperLink>
+        <Breadcrumbs
+          path={"educational"}
+          suffix=""
+          title={"SSW Consulting Educational - Sydney, Brisbane, Melbourne"}
+        />
```
Comment (bradystroud) [inline pages/educational/index.tsx]: These hardcoded strings are not ideal.

Copy the approach used by the consulting page template

https://github.com/SSWConsulting/SSW.Website-v3/blob/0cc5673ad1ca0ac88daba4ce2bea45b882184082/pages/consulting/[filename].tsx#L62

File: pages/educational/index.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            {buttonText}
+          </Button>
+        </a>
+      </div>
+    ),
+  };
+
+  return (
+    <Layout>
+      <SEO seo={{ ...node.seo, canonical: "/educational" }} />
```
Comment (bradystroud) [inline pages/educational/index.tsx]: Check out
https://github.com/SSWConsulting/SSW.Website-v3/blob/main/pages/consulting/[filename].tsx#L202

File: .tina/collections/educational.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      type: "string",
+      label: "Placeholder",
+      name: "placeholder",
+      ui: {
+        component: () => (
+          <p>
+            This is a placeholder block for
+            <span className="font-bold">Consulting Enquiry Form</span>
+          </p>
+        ),
```
Comment (bradystroud) [inline .tina/collections/educational.tsx]: image

**Review by bradystroud — CHANGES_REQUESTED** [review]
As per our conversation, there are a few changes 🙂

File: public/images/icons/arrow-right.png
Comment (jimmidier) [reply public/images/icons/arrow-right.png:1]: This icon is being used at https://github.com/SSWConsulting/SSW.Website-v3/blob/299-page-educational/tailwind.config.js#L330, thus not removed

**Review by bradystroud — APPROVED** [review]

**Pull request merged**